### PR TITLE
DROTH-2524 Additional signs with time range

### PR DESCRIPTION
**Pull request by marquesfm**
- Fix text position of the valid period additional signs;
- Create new validation for valid period additional signs form.

### CODE REVIEW

File: UI/src/assetTypeConfiguration.js [hunk 1]
```diff
@@ -1393,10 +1411,32 @@
...
+          if (_.isEmpty(additionalPanelsProps.values)) {
+            isValidAdditionalSign = true;
+          } else {
+            isValidAdditionalSign = _.every(additionalPanelsProps.values, function (additionalPanelProp) {
+              var additionalSignsValidations = _.find(validations, function (validation) {
+                return _.includes(validation.types, additionalPanelProp.panelType);
+              });
+
+              var additionalPanelValue = additionalPanelProp.panelValue;
+              var isValidPanelValue = additionalSignsValidations ? additionalSignsValidations.validate(additionalPanelValue, additionalSignsValidations.regexToUse) : true;
```
Comment (RobertVisan) [inline UI/src/assetTypeConfiguration.js:1431]: We can return the value here.

**Review by RobertVisan — APPROVED** [review]

File: UI/src/assetTypeConfiguration.js [hunk 2]
```diff
@@ -1393,10 +1411,32 @@
...
+
+              return isValidPanelValue;
+            });
+          }
```
Comment (FilPinto) [inline UI/src/assetTypeConfiguration.js:1435]: I don't think you need to do a if else here. Just start as the validation to true and then if you have additional panels do the the validation to change that boolean 

```javascript
var isValidAdditionalSign = true;
...
if (!_.isEmpty(additionalPanelsProps.values)) {
...
}
```

File: UI/src/assetTypeConfiguration.js (hunk 1 above)
Comment (marquesfm) [reply UI/src/assetTypeConfiguration.js:1431]: Done

File: UI/src/assetTypeConfiguration.js (hunk 2 above)
Comment (marquesfm) [reply UI/src/assetTypeConfiguration.js:1435]: Done

**Pull request merged**